lib/image-diff.js

```diff
@@ -202,7 +205,7 @@ ImageDiff.prototype = {
         fs.unlink(filepath, cb);
       }, function callOriginalCallback (_err) {
         // Callback with the imagesAreSame
-        callback(err, imagesAreSame);
+        callback(err, imagesAreSame, diffPath);
```

Comment:
To keep the API as simple as possible for future changes, please don't send back the `diffPath`. The user already knows what the is since they gave it to us.